app/src/main/java/com/futurice/freesound/app/FreesoundApplication.java

```diff
@@ -72,4 +74,11 @@ private void initStetho() {
                       .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
                       .build());
     }
+
+    private void initLeakCanary() {
+        if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
This was actually just copied from the LeakCanary sample code, but good point it does look a little odd.. I'll change it.